src/modules/background.js

```diff
@@ -0,0 +1,77 @@
+import { creator } from './helpers';
+import humLight from '../assets/images/humidity_light.png';
+import windSLight from '../assets/images/wind_speed_light.png';
+import humDark from '../assets/images/humidity_dark.png';
+import windSDark from '../assets/images/wind_speed_dark.png';
+import img1D from '../assets/images/01d.png';
+import img1N from '../assets/images/01n.png';
+import img2D from '../assets/images/02d.png';
+import img2N from '../assets/images/02n.png';
+import img3D from '../assets/images/03d.png';
+import img3N from '../assets/images/03n.png';
+import img4D from '../assets/images/04d.png';
+import img4N from '../assets/images/04n.png';
+import img9D from '../assets/images/09d.png';
+import img9N from '../assets/images/09n.png';
+import img10D from '../assets/images/10d.png';
+import img10N from '../assets/images/10n.png';
+import img11D from '../assets/images/11d.png';
+import img11N from '../assets/images/11n.png';
+import img13D from '../assets/images/13d.png';
+import img13N from '../assets/images/13n.png';
+import img50D from '../assets/images/50d.png';
+import img50N from '../assets/images/50n.png';
+
+const iconList = {
+  '01d': img1D,
+  '01n': img1N,
+  '02d': img2D,
+  '02n': img2N,
+  '03d': img3D,
+  '03n': img3N,
+  '04d': img4D,
+  '04n': img4N,
+  '09d': img9D,
+  '09n': img9N,
+  '10d': img10D,
+  '10n': img10N,
+  '11d': img11D,
+  '11n': img11N,
+  '13d': img13D,
+  '13n': img13N,
+  '50d': img50D,
+  '50n': img50N,
+};
+
+function createBGContainer(main) {
```

Comment:
**Let's make use of the arrow function instead of using the `function` keyword**
![image](https://user-images.githubusercontent.com/31897434/92687892-3b7b3280-f334-11ea-860a-cf5e33ef18f8.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/